Neos.Flow/Classes/Http/Response.php

```diff
@@ -20,6 +20,7 @@
  *
  * @api
  * @Flow\Proxy(false)
+ * @deprecated
```

Comment:
```suggestion
 * @deprecated Since Flow 5.3, to be removed with 6.0. The whole Flow HTTP implementation will use PSR-7 compatible implementations only from then.
```